proxy/src/proxy.rs

```diff
@@ -391,16 +391,11 @@ fn handle_server_event(
                 .authenticate_continue(continuation_request)
                 .unwrap();
         }
-        ClientFlowEvent::AuthenticateAccepted { status, .. } => {
+        ClientFlowEvent::AuthenticateStatusReceived { status, .. } => {
             trace!(response=%format!("{:?}", status).blue(), role = "s2p", "<--| Received authentication accepted");
```

Comment:
You forgot to modify the log message.

Reply:
I changed it now but rebasing on #141 will change again.